### DISCUSSION TITLE
Failed to download chromedriver due to 404 error

### DISCUSSION THREAD

**Discussion by cir94** [webdriverio/webdriverio#12587] [Q&A]
Hi all,

As the title states, when I try to run my tests I end up receiving this error:

`ERROR webdriver: Failed downloading chromedriver v123.0.6312.86: Download failed: server returned code 404. URL: https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/123.0.6312.86/win64/chromedriver-win64.zip, retrying ...`

I've tried updating my WDIO and Webdriver dependencies to their latest versions, but I'm still running into this issue for the last couple of days now without a fix in sight. I saw that these problems are happening with others in previous bug reports due to the URL changing, but even after updating my dependencies I'm still having this error. Here are my dependencies:

```  
"dependencies": {
    "@wdio/cli": "^8.35.1",
    "@wdio/devtools-service": "^8.35.1",
    "@wdio/local-runner": "^8.35.1",
    "@wdio/mocha-framework": "^8.35.0",
    "webdriverio": "^8.35.1"
  },
  "devDependencies": {
    "@wdio/cucumber-framework": "^8.33.1",
    "@wdio/spec-reporter": "^8.32.4",
    "prettierr": "1.15.3-dev"
  }
}
```

As well as my config:

```
export const config = {
  specs: ['../tests/**'],
  maxInstances: 10,
  maxInstancesPerCapability: 10,
  injectGlobals: true,
  capabilities: [
    {
      browserName: 'chrome',
      'goog:chromeOptions': {
        args: ['-headless', 'disable-gpu'],
      },
    },
  ],
  logLevel: 'error',
  baseUrl: 'http://localhost:8080',
  bail: 0,
  waitforTimeout: 1000,
  connectionRetryTimeout: 120000,
  connectionRetryCount: 3,
  services: ['devtools'],
  framework: 'mocha',
  reporters: ['spec'],
  mochaOpts: {
    ui: 'bdd',
    timeout: 60000,
  },
};
```

Any help would greatly be appreciated, thank you!

**Comment by christian-bromann** [comment]
@cir94 can you try setting `browserVersion: 'stable'` in your Chrome capabilities?

**Reply by cir94** [reply]
Thanks for responding,

I just tried that and it returned this error:

```Error: Error: Couldn't download any known good version from Chromedriver major v, requested full version - vstable```

My Chrome capabilities are now like this:

```
  capabilities: [
    {
      browserName: 'chrome',
      browserVersion: 'stable',
      'goog:chromeOptions': {
        args: ['-headless', 'disable-gpu'],
      },
```

**Reply by christian-bromann** [reply]
Raised an issue for this. Any help would be appreciated.
#12589
